DEVELOPER_GUIDE.md

```diff
@@ -184,6 +186,25 @@ Additionally, it is possible to attach one debugger to the cluster JVM and anoth
 ./gradlew :integTest -Dtest.debug=1 -Dcluster.debug=1
 ```
 
+## Backwards Compatibility Testing
```

Comment:
as per my understanding for Apple silicon chips and even for intel chips the BWC will not run. Can we add these details in the Developer guide.